apps/taxes/src/pages/providers/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { Box, Button, Text } from "@saleor/macaw-ui/next";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import { AvataxIcon, StripeTaxIcon, TaxJarIcon } from "../../assets";
+import { AppCard } from "../../modules/ui/app-card";
+import { AppColumns } from "../../modules/ui/app-columns";
+
+const Header = () => {
+  return (
+    <Box>
+      <Text __maxWidth={"360px"} __fontWeight={"400"} variant="body">
```

Comment:
fontWeight should use what Macaw provides (no `__`, use allowed values)

Reply:
Unfortunately, I have some issues with Macaw font weights. I will describe them to Macaw team.